### PR TITLE
Sync daycare from pokecrystal

### PR DESCRIPTION
**Pull request by rtandy**
Mostly a straight copy from pokecrystal. Minor difference since there is no Odd Egg event, the logic for talking about Eggs is shared between the two NPCs thus I merged/renamed the corresponding `_ACTIVE_F` flags.

The name `SFX_GET_EGG_FROM_DAYCARE_LADY` seems slightly odd to me. AFAIK, in game one never receives an egg from the Lady, only from the Aide and the Man. It looks like SFX `0x95` and `0x96` point to the same SFX, and indeed they sound the same in game. Searching in pokecrystal, `0x95` doesn't appear to be used; the Aide and the Day-care Man both use `SFX_GET_EGG_FROM_DAYCARE_LADY`. Gold doesn't appear to have the ReceiveTogepiEggScript disassembled yet but I would assume it's the same.

Anyway basically I wonder if those should be renamed to `SFX_GET_EGG_UNUSED` (or `SFX_UNKNOWN_95`) and `SFX_GET_EGG` or something like that? That would apply to pokecrystal as well of course.

### CODE REVIEW

**Comment by iimarckus** [comment]
Very nice, thanks.

Renaming the unused sound effect seems fine.
